general/development/process/release.md

```diff
@@ -100,6 +100,7 @@ tags:
 | 11. | &#10003; |  | Upgrade [moodle.org](https://moodle.org/) to beta release |  |
 | 12. | &#10003; |  | Prepare pull requests for CI Repositories:<ul><li>`MR1`: Create a new version, with unit tests, in Jobs repository ([see example from 3.11 release](https://git.in.moodle.com/integration/nightlyjobs/merge_requests/24))</li><li>`MR2`: Create a new symlink for the version in [moodle-ci-runner](https://github.com/moodlehq/moodle-ci-runner) repository ([see example from 3.11 release](https://github.com/moodlehq/moodle-ci-runner/pull/41))</li><li>`MR3`: Configure CIs to skip language upgrade in [moodle-ci-runner](https://github.com/moodlehq/moodle-ci-runner) repository ([see example from 3.10 release](https://github.com/moodlehq/moodle-ci-runner/pull/35))</li><li>`MR4`: Configure CIs to stop skipping language upgrade in [moodle-ci-runner](https://github.com/moodlehq/moodle-ci-runner) repository for after on-sync ([see example from 3.10 release](https://github.com/moodlehq/moodle-ci-runner/pull/40)) -- **Note**: This change should be based on the previous and only applied when the lang packs corresponding to the version being released are available.</li></ul> | Integration Team |
 | 13. | &#10003; |  | Go through all points listed under 1 week prior in [New docs version process](/general/development/process/release/newuserdocs). | Community Manager |
+| 14. | &#10003; |  | Update the `BRANCHLIST` and `VERSIONLIST` in the [JSDoc](https://github.com/moodle/jsdoc) and [PHPDoc](https://github.com/moodle/phpdoc) repositories with the list of supported versions. These are built on a Sunday so this should be done in the final week before release. | Integration Team |
```

Comment:
```suggestion
| 14. | &#10003; | &#10003; | Update the `BRANCHLIST` and `VERSIONLIST` in the [JSDoc](https://github.com/moodle/jsdoc) and [PHPDoc](https://github.com/moodle/phpdoc) repositories with the list of supported versions. These are built on a Sunday so this should be done in the final week before release. | Integration Team |
```

Reply:
As we also need to remove unsupported versions from the list, shouldn't this action be checked also before minors (as we do with some similar maintenance tasks related to unsupported branches?)

Reply:
Yup - you're right.